salt/modules/yumpkg.py

```diff
@@ -14,6 +14,8 @@
 
 .. versionadded:: 3003
     Support for ``tdnf`` on Photon OS.
+.. versionadded:: 3007.0
+    Support for ``dnf5``` on Fedora 39
```

Comment:
Is the support only for Fedora 39? Isn't dnf5 going to be applied to other OSs in the future as well?

Reply:
I expect it will be applied to other OS's in the future, RHEL 9 yum is really dnf under the very thin covers (RHEL 8 yum was yum), and hence I would expect to see these changes appearing in RHEL 10 whenever it is released.